hass_nabucasa/cloud_api.py

```diff
@@ -208,12 +208,9 @@ async def async_files_download_details(
     if TYPE_CHECKING:
         assert cloud.id_token is not None
     resp = await cloud.websession.get(
-        f"https://{cloud.servicehandlers_server}/files/download_details",
+        f"https://{cloud.servicehandlers_server}/files"
+        f"/download_details/{storage_type}/{filename}",
```

Comment:
Is this correct? Does the second string do anything?

Reply:
This is correct, it is a single string when passed like this, and this is split to deal with length limits.